### DISCUSSION TITLE
How to chagne notification size?

### DISCUSSION THREAD

**Discussion by maguro1kan** [AvaloniaCommunity/Prism.Avalonia#209] [Q&A] [answered]
How to chagne notification size?

![image](https://github.com/user-attachments/assets/82b5a680-a7fa-4250-8ae5-b59ff5ccc626)

and I want to remove NotificationType

Thanks

**Comment by DamianSuess** [comment, marked as answer]
This is a feature is a part of Avalonia itself and is currently not sizable. Feel free to create a feature request over there
